keras_cv/models/stable_diffusion/stable_diffusion.py

```diff
@@ -371,6 +390,14 @@ def _get_unconditional_context(self):
 
         return unconditional_context
 
+    def _expand_tensor(self, text_embedding, batch_size):
```

Comment:
maybe put a docstring here as well, even it's a private method

Reply:
Done! Let me know if this is alright.

I added this method so we didn't have to repeat the same code when computing the `unconditional_context`.